lms/static/my.js

```diff
@@ -16,6 +16,19 @@ function escapeUnicode(str) {
 }
 
 
+function copyCode(button) {
+  button.addEventListener('click', () => {
+    var copyText = document.getElementById('python-user-code');
```

Comment:
Use `let` or `const`, not `var`. Also, if possible, define the variables as soon as possible.